src/adapters/appnexusAst.js

```diff
@@ -193,8 +195,11 @@ function AppnexusAstAdapter() {
       if (type === 'native') bid.mediaType = 'native';
       if (type === 'video') bid.mediaType = 'video';
       if (type === 'video-outstream') bid.mediaType = 'video-outstream';
-      const placement = bidRequests[bid.adId].placementCode;
-      bidmanager.addBidResponse(placement, bid);
+
+      if (bid.adId in bidRequests) {
```

Comment:
yeah that makes sense. In that case we are OK to throw away the bid response so this is ok.